wp-react-blocks-plugin/blocks/charts/BlockEdit.js

```diff
@@ -99,7 +99,7 @@ class BlockEdit extends BaseBlockEdit {
                                             value: 'numberVarietiesSold'
                                         },
                                         {
-                                            label: 'Efficiency of seed import process',
+                                            label: 'Number of Seed Inspectors and Industry Opinion Rating',
```

Comment:
Please leave it as 'Efficiency of seed import process' (the original name was wrong).

Reply:
done